app/build.gradle.kts

```diff
@@ -72,7 +72,6 @@ dependencies {
 	implementation(libs.android.compose.activity)
 	implementation(libs.android.compose.navigation)
 	implementation(libs.android.google.oss.licenses)
-	implementation(libs.android.google.services)
```

Comment:
That was added [here](https://github.com/intive/Picover/pull/50/files#diff-8cff73265af19c059547b76aca8882cbaa3209291406f52df1dafbbc78e80c46R52), redundantly and was causing an error 😢